src/services/expense.service.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const { getId } = require('./getId');
+
+const expenses = [];
+
+const init = () => {
+  expenses.length = 0;
+};
+
+const getAll = (params) => {
+  let filteredExpenses = expenses;
```

Comment:
better use const here (optional) (not critical)

Reply:
Agreed. 
Why do we need let in this case? 
Don`t see any reason why we don't use const for this array.
Try rewriting using const. We avoid not needed assignments.